lib/core/config.rb

```diff
@@ -71,6 +76,57 @@ def ensure_config_app!(app_name, app_options)
     raise "App '#{app_name}' is empty in 'controlplane.yml'." unless app_options
   end
 
+  def ensure_org!
+    return if @org
+
+    raise "No org provided. The org can be provided either through --org " \
+          "or the CPLN_ORG env var ('allow_org_override_by_env' must be set to true in 'controlplane.yml')."
+  end
+
+  def ensure_app!
+    return if @app
+
+    raise "No app provided. The app can be provided either through --app " \
+          "or the CPLN_APP env var ('allow_app_override_by_env' must be set to true in 'controlplane.yml')."
+  end
+
+  def ensure_required_options!
+    missing_str = required_options
+                  .reject { |option_name| %i[org app].include?(option_name) || options.key?(option_name) }
+                  .map { |option_name| "--#{option_name}" }
+                  .join(", ")
+
+    raise "Required options missing: #{missing_str}" unless missing_str.empty?
+  end
+
+  def non_empty_str_or_nil(str)
```

Comment:
To make sure that `config.org` and `config.app` will always be either `nil` or a non-empty string, never an empty string.

Reply:
Hm, I don't know, I usually prefer being very explicit about method names, because then we don't even need to go to the method to know what it does / outputs. `non_empty_str_or_nil` tells me I'll get either a non-empty string or `nil`, while `sanitize` doesn't tell me anything about what it's doing.

But doing `strip` inside the method sounds like a good idea, since we're already doing that every time we call it anyway.

Reply:
Renamed it to `strip_str_and_validate`, what do you think about this name?